hummingbot/connector/exchange/paper_trade/paper_trade_exchange.pyx

```diff
@@ -1036,6 +1045,9 @@ cdef class PaperTradeExchange(ExchangeBase):
         else:
             return Decimal(f"1e-10")
 
+    def get_order_price_quantum(self, trading_pair: str, price: Decimal) -> Decimal:
+        return self.c_get_order_price_quantum(trading_pair, price)
+
```

Comment:
Is this part of the config map migration?

Reply:
I think I've originally added that for debugging, but without it, `get_order_price_quantum` is undefined